### PR TITLE
feat: Microblock details hints

### LINKED ISSUE #373: Microblock details hints
### Please describe the problem that your request should resolve.
Implement hints on the Microblock details page similarly to how it's done on other entity details pages.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #373 

## Demo

https://github.com/aeternity/aescan/assets/15363559/5fd7e31e-88e7-4245-a357-8192dfb2e566

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change requires a change to the documentation.
- [x] add microblocks hints

### CODE REVIEW

File: src/utils/hints/microblocksHints.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const microblocksHints = {
+  microblock: 'Microblocks can be created every 3 seconds by the current leader, are linked to a specific keyblock and include all types of on-chain transactions. The next leader is incentivized to keep the microblocks in the blockchain as he receives 40% of the fees of the transactions.',
+  keyblock: 'Parent keyblock id in which the microblock is linked.',
+  microblockHeight: 'Parent keyblock height in which the microblock is linked.',
```
Comment (michele-franchi) [inline src/utils/hints/microblocksHints.js:4]:
```suggestion
  microblockHeight: 'Parent keyblock height to which the microblock is linked.',
```

Looks better to me

File: src/utils/hints/microblocksHints.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const microblocksHints = {
+  microblock: 'Microblocks can be created every 3 seconds by the current leader, are linked to a specific keyblock and include all types of on-chain transactions. The next leader is incentivized to keep the microblocks in the blockchain as he receives 40% of the fees of the transactions.',
+  keyblock: 'Parent keyblock id in which the microblock is linked.',
```
Comment (michele-franchi) [inline src/utils/hints/microblocksHints.js:3]:
```suggestion
  keyblock: 'Parent keyblock id to which the microblock is linked.',
```

Looks better to me

**Review by michele-franchi — APPROVED** [review]
I have two minor suggestions but it looks good!

File: src/utils/hints/microblocksHints.js (hunk 2 above)
Comment (janmichek) [reply src/utils/hints/microblocksHints.js]: fixed

**Review by lukeromanowicz — APPROVED** [review]
There is nothing to add. Looks good to me too :) Well done!

File: src/utils/hints/microblocksHints.js (hunk 1 above)
Comment (michele-franchi) [reply src/utils/hints/microblocksHints.js:4]: This was marked as resolve but I still see `in`

**Review by michele-franchi — APPROVED** [review]
Looks good, please have a look at my comment

**Comment by lukeromanowicz** [comment]
The page title hints have been just merged. I think it will impact the code in this PR

**Comment by janmichek** [comment]
> The page title hints have been just merged. I think it will impact the code in this PR

conflict resolved

File: src/utils/hints/microblocksHints.js (hunk 1 above)
Comment (janmichek) [reply src/utils/hints/microblocksHints.js]: Sorry, my bad. Fixed

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**